recipes/proplot/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "proplot" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 75e6c871bb628ed2879eb1a997c159a405ba69655e8010489f50f74877813b7a
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - rm -rf proplot/fonts
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools_scm
+    - setuptools_scm_git_archive
+    - setuptools >=30.3.0
+  run:
+    - python >=3
+    - matplotlib-base
+    - lxml
+    # https://github.com/lukelbd/proplot/issues/86
+    - pyyaml
+
+test:
+  imports:
+    - proplot
+
+about:
+  home: https://proplot.readthedocs.io/en/latest/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'A comprehensive, easy-to-use matplotlib wrapper for making beautiful, publication-quality graphics.'
+  dev_url: https://github.com/simplejson/simplejson
```

Comment:
I think the `dev_url` should be https://github.com/lukelbd/proplot?

Reply:
Thanks!